### ISSUE TITLE
[A11y][TFM]Narrator/NVDA is not announcing the state for the control 'shows and hides TFM filters' 

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9412]
Url: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1751374

Environment Details:
Application Name: Filtering by Target Frameworks
#URL: [NuGet Gallery | Packages (nugettest.org)](https://dev.nugettest.org/packages?q=)
Windows Version: Win11
OS Version: 22621.1105
Chromium Edge Version: Version 109.0.1518.78 (Official build) (64-bit)
Chrome Version: Version 110.0.5481.77 (Official Build) (64-bit)
Screen reader version: Narrator/ NVDA 2022.2

Repro Steps: 
Hit the URL: [NuGet Gallery | Packages (nugettest.org)](https://dev.nugettest.org/packages?q=)
Tab to Frameworks section.
Tab to the collapse/expand buttons present in the Frameworks section.
Turn on Narrator/NVDA.
Observe that screen reader is not announcing the state whether collapsed/expanded or hidden/shown for the control 'shows and hides TFM filter'.

Actual Result:
Narrator/NVDA is not announcing the state for the control 'shows and hides TFM filters'  whether it is collapsed/expanded or hidden/shown.
Expected Result:
Narrator/NVDA should announce the state for the control 'shows and hides TFM filters' whether it is collapsed/expanded or hidden/shown.

Notes:
The expander on the TFM filters doesn't appear to be marked as a control with a state. We have other custom expander controls on other pages and should be able to fix this by replicating the aria state on those controls.
